plugins/hls-fourmolu-plugin/src/Ide/Plugin/Fourmolu.hs

```diff
@@ -75,6 +81,11 @@ provider recorder plId ideState typ contents fp fo = ExceptT $ withIndefinitePro
             (pure . Left . PluginInternalError . T.pack . show)
              $ runExceptT $ cliHandler fileOpts
         else do
+#if MIN_VERSION_fourmolu(0,12,0)
+            logWith recorder Info $ LogBuiltinVersion $ Just Fourmolu.version
```

Comment:
Is this better than `read #VERSION_fourmolu`? That should work all the time, and for all packages, right?

Reply:
Thanks! I never know this before!😅